compiler-rt/include/sanitizer/rtsan_interface.h

```diff
@@ -0,0 +1,108 @@
+//===-- sanitizer/rtsan_interface.h -----------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file is a part of RealtimeSanitizer.
+//
+// Public interface header.
+//===----------------------------------------------------------------------===//
+
+#ifndef SANITIZER_RTSAN_INTERFACE_H
+#define SANITIZER_RTSAN_INTERFACE_H
```

Comment:
It's my thought that we can point people who aren't using llvm 20 to this file and it "just works" -- we discussed rolling one of these files but this actually makes it more official. Everything in here has been designed to match the other sanitizer, plus be able to work independently